src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -108,10 +113,32 @@ public void addPerson(Person person) {
     @Override
     public void setPerson(Person target, Person editedPerson) {
         requireAllNonNull(target, editedPerson);
-
         addressBook.setPerson(target, editedPerson);
     }
 
+    @Override
+    public void addGroup(Group group) {
+        addressBook.addGroup(group);
+        updateFilteredPersonList(x->group.getPersons().contains(x));
```

Comment:
I think there should be a space between the arrow heads. `updateFilteredPersonList(x -> group.getPersons().contains(x))`